rest-service/manager_rest/syncthing_status_manager.py

```diff
@@ -0,0 +1,90 @@
+from datetime import datetime, timedelta
+
+from flask import current_app
+
+from cloudify.cluster_status import (ServiceStatus,
+                                     NodeServiceStatus)
+
+from manager_rest.storage import models, get_storage_manager
+from manager_rest.rest.rest_utils import parse_datetime_string
+
+try:
+    from cloudify_premium import syncthing_utils
+    from cloudify_premium.ha import utils as ha_utils
+except ImportError:
+    syncthing_utils = None
+    ha_utils = None
+
+
+def _last_manager_in_cluster():
+    storage_manager = get_storage_manager()
+    managers = storage_manager.list(models.Manager,
```

Comment:
probably `.full_access_list` because we don't need the pagination or the tenant stuff here. Except that full_access_list of course doesn't have a `sort` so maybe add that. Or feel free to not sort.

Reply:
Hmm. I'm not sure this one is worth rewriting since pagination would only take effect if the manager count surpassed the page size so probably not worth mucking about with as this was just copied code?